datatypes-cose/src/commonMain/kotlin/at/asitplus/crypto/datatypes/cose/CoseAlgorithm.kt

```diff
@@ -37,23 +37,23 @@ enum class CoseAlgorithm(val value: Int) {
 
 
     fun toCryptoAlgorithm() = when(this) {
```

Comment:
But now the function name does not match the class name of the return values

Reply:
renamed